index.js

```diff
@@ -1,10 +1,13 @@
 module.exports = {
   root: true,
   env: {
+    browser: true,
```

Comment:
I don't think this should reference browser. I see there are a couple of rules here targeting react / front-end. For those cases we should add the rules to eslint-config-react and extend from there as well (or have a dedicated preset for react that extends the default one and archive the other repo)

Reply:
Removed here: https://github.com/open-turo/eslint-config-typescript/pull/118/commits/5aa24f91e46f3f2717aea032e4c97357e756228f
And added here: https://github.com/open-turo/eslint-config-react/pull/85